SETUP.md

```diff
@@ -7,27 +7,28 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 
 ## Overview
 
-1. Clone the repo, which also may take a while.
-    - The simplest option is to clone via SSH with: `git clone git@github.com:code-dot-org/code-dot-org.git`
-    - The fastest option is to clone via HTTP with: `git clone https://github.com/code-dot-org/code-dot-org.git`. Although faster than SSH, this option requires you to reauthenticate every time you want to update. You will therefore probably want to switch to SSH after the initial clone with `git remote set-url origin git@github.com:code-dot-org/code-dot-org.git`
-
-1. `cd code-dot-org`
-
```

Comment:
These steps have been moved below "Install OS-specific pre-requisites" because Git LFS should be installed before cloning.